ddtrace/opentelemetry/_utils.py

```diff
@@ -0,0 +1,52 @@
+from functools import partial
+
+from opentelemetry.context.context import Context
+from opentelemetry.trace.propagation import get_current_span
+from opentelemetry.trace.propagation import set_span_in_context
+from opentelemetry.trace.span import Span as OtelSpan
+
+import ddtrace
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.internal.wrapping import wrap
+from ddtrace.opentelemetry.span import Span
+
+
+_DDOTEL_SPAN_KEY = "dd_span"
+
+
+def _ddtrace_get_current_span(ddtracer, func, args, kwargs):
+    otel_context = get_argument_value(args, kwargs, 0, "context")
+    if otel_context:
+        return otel_context.get(_DDOTEL_SPAN_KEY, None)
+
+    ddactive = ddtracer.context_provider.active()
+    if ddactive is None or isinstance(ddactive, ddtrace.context.Context):
+        # There is not active span if the ddtrace provider returns a DDContext object
```

Comment:
FIXME: if `isinstance(ddactive, ddtrace.context.Context) is True` return a `NonRecordingSpan()`